anchore_engine/services/apiext/api/controllers/system.py

```diff
@@ -376,3 +378,101 @@ def describe_error_codes():
         httpcode = return_object['httpcode']
 
     return return_object, httpcode
+
+
+@authorizer.requires([])
```

Comment:
I think only the admin should be able to run this command, since it creates output to an external system and it's the admin that configures the webhooks in the config.yaml.

You can enforce that with:
`@authorizer.requires_account(with_names=[localconfig.ADMIN_ACCOUNT_NAME])`

That will make it only authorized for users that are in the 'admin' account.

Reply:
fixed